test/runtests.jl

```diff
@@ -13,6 +13,8 @@ using Test
 using SparseArrays
 using InfiniteArrays
 
+include("aqua.jl")
```

Comment:
I find that having a separate file makes it easier to run only the Aqua tests, as one can just include the file instead of running the full testset. This makes fixing Aqua-related issues smoother. However, this is just a stylistic choice, and I'm happy to include it in the same file. I'd still prefer this to be at the top rather than the bottom, as it's better to run this before too many other packages are loaded that may introduce ambiguities etc.

Reply:
Updated to only have a single file